patches/server/0028-Reset-large-tags.patch

```diff
@@ -0,0 +1,249 @@
++import java.nio.charset.StandardCharsets;
++
++public class NbtUtility
++{
++
++    public static boolean isTooLarge(@Nullable CompoundTag tag)
++    {
++        if (tag == null) return false;
++        return tag.toString().getBytes(StandardCharsets.UTF_8).length > (256 * 1024);
++    }
++
++    public static class Item
++    {
++
++        public static CompoundTag removeItemData(CompoundTag tag)
++        {
++            CompoundTag cleaned = new CompoundTag();
++            cleaned.putString("id", tag.getString("id"));
++            cleaned.putByte("Count", tag.getByte("Count"));
++            return cleaned;
++        }
++    }
++}
+diff --git a/src/main/java/net/minecraft/world/item/ItemStack.java b/src/main/java/net/minecraft/world/item/ItemStack.java
+index d02339a07e1da56fd5dc171a5e32b00d63fa555c..fe6cb52573ee50a2c02d7add38b5e8411d7177f4 100644
+--- a/src/main/java/net/minecraft/world/item/ItemStack.java
++++ b/src/main/java/net/minecraft/world/item/ItemStack.java
+@@ -1,5 +1,6 @@
+ package net.minecraft.world.item;
+
++import com.github.atlasmediagroup.scissors.NbtUtility;
+ import com.google.common.collect.HashMultimap;
+ import com.google.common.collect.Lists;
+ import com.google.common.collect.Multimap;
+@@ -7,23 +8,28 @@ import com.google.gson.JsonParseException;
+ import com.mojang.brigadier.StringReader;
+ import com.mojang.brigadier.exceptions.CommandSyntaxException;
+ import com.mojang.serialization.Codec;
++import com.mojang.serialization.Dynamic;
+ import com.mojang.serialization.codecs.RecordCodecBuilder;
+-import java.text.DecimalFormat;
+-import java.text.DecimalFormatSymbols;
+-import java.util.Collection;
+-import java.util.Iterator;
+-import java.util.List;
+-import java.util.Collections;
+-import java.util.Comparator;
+-import java.util.Locale;
+-import java.util.Map.Entry;
+-import java.util.Objects;
+-import java.util.Optional;
+-import java.util.Random;
+-import java.util.function.Consumer;
+-import java.util.function.Predicate;
+-import java.util.stream.Collectors;
+-import javax.annotation.Nullable;
++import net.minecraft.ChatFormatting;
++import net.minecraft.Util;
++import net.minecraft.advancements.CriteriaTriggers;
++import net.minecraft.commands.arguments.blocks.BlockPredicateArgument;
++import net.minecraft.commands.arguments.blocks.BlockStateParser;
++import net.minecraft.core.BlockPos;
++import net.minecraft.core.Direction;
++import net.minecraft.core.Registry;
++import net.minecraft.nbt.CompoundTag;
++import net.minecraft.nbt.ListTag;
++import net.minecraft.nbt.NbtOps;
++import net.minecraft.network.chat.*;
++import net.minecraft.network.protocol.game.ClientboundBlockUpdatePacket;
++import net.minecraft.resources.ResourceLocation;
++import net.minecraft.server.MinecraftServer;
++import net.minecraft.server.level.ServerLevel;
++import net.minecraft.server.level.ServerPlayer;
++import net.minecraft.sounds.SoundEvent;
++import net.minecraft.sounds.SoundSource;
++import net.minecraft.stats.Stats;
```

Comment:
I think your IDE may have done a little thing there

Reply:
Mm probably but it works fine